planemo/autoupdate.py

```diff
@@ -78,6 +79,18 @@ def get_tokens(xml_tree: ElementTree) -> Dict[str, Dict[str, Optional[str]]]:
     return tokens
 
 
+def parse_version(version: str) -> packaging.version.Version:
+    """
+    Parse a version string into a Version object.
+    If the version string is not PEP 440 compliant, return Version("0").
+    """
+    try:
+        return packaging.version.parse(version)
```

Comment:
Wouldn't it make more sense to use our vendored copy of packaging.version ?

Reply:
Good call!